src/components/ExtendedReminder/index.tsx

```diff
@@ -127,19 +128,25 @@ const ExtendedReminder: FunctionComponent<ExtendedReminderProps> = ({
   } as const
 
   return (
-    <StyledContainer variant={variant} {...props}>
+    <StyledContainer
+      variant={variant}
+      aria-label={variant}
```

Comment:
I agree, sorry it's useless because the same text is already in context.

I do not remember why I asked for it 🤔

Reply:
a11y is failing when I remove `aria-label`, seems like it's needed to distinct elements: https://github.com/scaleway/scaleway-ui/runs/4605195641?check_suite_focus=true#step:5:83

Reply:
@QuiiBz I Think you can remove event aria related stuffs here : role region in StyledContainer, role status in StyledTitle and aria labels. It is useless because theses reminders does not pop if I'm not wrong

Reply:
Alright thanks!